### DISCUSSION TITLE
유니티로 빌드잇 여는 법?

### DISCUSSION THREAD

**Discussion by K2Ken** [naverz/zepeto-studio-kor#221] [5. Q&A]
안녕하새요 제페토빌드잇 맵을 유니티로 열고 시픈데 어떡해 할지 모르겠어요.

![image](https://user-images.githubusercontent.com/97222267/149644956-455ec46c-c350-4d9e-9e71-17d865476013.png)
![image](https://user-images.githubusercontent.com/97222267/149645031-551457dd-5ec2-43b9-bc3e-29340187d88d.png)

제페토 가이드를 보아도 무슨 뜻 인지 모르겠어요, 혹시 저 빌드잇 칭을 어떡해 여는건지 아시는 분은 댓글에 남겨 주세요.

**Comment by junekinns** [comment]
<img width="1065" alt="Screen Shot 2022-01-16 at 3 01 58 PM" src="https://user-images.githubusercontent.com/28533936/149649236-044c34b4-896b-4c70-b468-0f1dbd93c646.png">
스크린샷처럼 하시면 됩니다.
